package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hmcts/ccd-case-ui-toolkit",
-  "version": "4.18.19-specific-access-reason-merged",
+  "version": "4.18.19-Fix-old-case-id",
```

Comment:
very minor comment - prefer all lower case letters as stops confusion when people use the tag